linux/Jenkinsfile

```diff
@@ -286,10 +286,59 @@ def uploadAlpineArtifacts(String buildArch) {
     // currently only support x64 as buildArch
     // set BUILDARCH environment variable for ${buildArch}
     env.BUILDARCH = buildArch
-    jf 'rt u **/build/ospackage/temurin-*j*.apk apk/alpine/main/${BUILDARCH}/ --flat=true'
-    jf 'rt u **/build/ospackage/temurin-*src*.apk apk/alpine/main/${BUILDARCH}/ --flat=true'
+
+    def PackFiles = findFiles(glob: '**/build/ospackage/temurin-*j*.apk') // List All Packages To Upload
+    def SrcFiles = findFiles(glob: '**/build/ospackage/temurin-*src*.apk') // List All Sources To Upload
+
+    for (PackFile in PackFiles) {
+
+      def FileName = PackFile.name
+      def Target = "https://adoptium.jfrog.io/artifactory/apk/alpine/main/${BUILDARCH}/${FileName}"
+
+      echo "File : ${PackFile}"
+      echo "Filename : ${FileName}"
+      echo "Target : ${Target}"
+
+      try {
```

Comment:
this feels like a lot of duplicated code. Can you split it into a function and reuse it?